src/modules/translation/languages/es.json

```
@@ -484,7 +484,8 @@
       },
       "failure": {
         "title": "Algo salió mal",
-        "subtitle": "No pudimos publicar tu escena"
+        "subtitle": "No pudimos publicar tu escena",
+        "subtitle_size_error": "The scene is too big to be published in a World.<br></br>The maximum size is <b>{maxSizeMbs} mbs</b>, but the current scene is <b>{deployedSizedMbs} mbs</b> large."
```

What do you think about using the MB / Mb notation for the size?
```suggestion
        "subtitle_size_error": "The scene is too big to be published in a World.<br></br>The maximum size is <b>{maxSizeMbs} mbs</b>, but the current scene is <b>{deployedSizedMbs} MBs</b> large."
```